blog/electron-36-0.md

```diff
@@ -0,0 +1,231 @@
+title: Electron 36.0.0
+date: 2025-04-28T00:00:00.000Z
+authors:
+  - georgexu99
+slug: electron-36-0
+tags: [release]
+---
+
+Electron 36.0.0 has been released! It includes upgrades to Chromium 136, V8 13.6, and Node 22.14.0.
+
+---
+
+The Electron team is excited to announce the release of Electron 36.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on [Bluesky](https://bsky.app/profile/electronjs.org) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Highlights
+
+- Add support for MacOS Autofill, Writing Tools (e.g., spelling, grammar), and Services
+  - Electron apps can now hook into AppKit's native context menus via the newly exposed `frame` option in `menu.popup`.
+- Added ServiceWorkerMain class to interact with service workers in the main process.
+  - Developers can now start serviceworkers with `startWorkerForScope()`, retrieve service worker instances with `fromVersionID()`, and listen for lifecycle events like `runnning-status-changed`
+
+### Stack Changes
+
+- Chromium `136.0.7103.48`
+  - [New in 136](https://developer.chrome.com/blog/new-in-chrome-136/)
+  - [New in 135](https://developer.chrome.com/blog/new-in-chrome-135/)
+- Node `22.14.0`
+  - [Node 22.14.0 blog post](https://nodejs.org/en/blog/release/v22.14.0/)
+- V8 `13.5`
+
+Electron 36 upgrades Chromium from `134.0.6998.23` to `136.0.7103.48`, and V8 from `13.5` to `13.6`.
+
+### New Features
+
+- Added `BrowserWindow.isSnapped()` to indicate whether a given window has been arranged via Snap. [#46226](https://github.com/electron/electron/pull/46226)
+- Added `ServiceWorkerMain` class to interact with service workers in the main process.
+  - Added `fromVersionID` on `ServiceWorkers` to get an instance of `ServiceWorkerMain`.
+  - Added `running-status-changed` event on `ServiceWorkers` to indicate when a service worker's running status has changed.
+  - Added `startWorkerForScope` on `ServiceWorkers` to start a worker that may have been previously stopped. [#45232](https://github.com/electron/electron/pull/45232) <span style="font-size:small;">(Also in [35](https://github.com/electron/electron/pull/45341))</span>
+- Added `WebContents.focusedFrame` to get the focused frame.
+  - Fixed `WebContents.opener` to specify potential `null` type. [#45667](https://github.com/electron/electron/pull/45667)
+- Added `contextBridge.executeInMainWorld` to safely execute code across world boundaries. [#45229](https://github.com/electron/electron/pull/45229) <span style="font-size:small;">(Also in [35](https://github.com/electron/electron/pull/45330))</span>
```

Comment:
We already mentioned this in the previous blog post. I believe we don't want to mention PRs that we've already covered in previous release blog posts.

There are more PRs like this. I haven't marked the remaining ones for now.